regen-network/regen-web
# chore: regenerate types from corrections on staging graphql

## Description

See changes in regen-sanity [here](https://github.com/regen-network/regen-sanity/pull/20/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R13)

Our sanity setup was configured to deploy data to production that was meant for staging, so the sanity types from my last PR weren't properly generated

regen-network/regen-sanity#20 (should be merged first), and it's probably smart to `import-production` in sanity before that. 

Closes: regen-network/regen-registry#1033
Closes: regen-network/regen-registry#843
Closes: regen-network/regen-registry#1017

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. after merging regen-network/regen-sanity#20, generate a new build and confirm `/team` shows with no errors

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
